Source/OxyPlot.DotNetCore/OxyPlot.DotNetCore.Drawing/GraphicsRenderContext.cs

```diff
@@ -0,0 +1,535 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.IO;
+using System.Linq;
+
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright filename="GraphicsRenderContext.cs" company="OxyPlot">
+//   The MIT License (MIT)
+//
+//   Copyright (c) 2012 Oystein Bjorke
+//
+//   Permission is hereby granted, free of charge, to any person obtaining a
+//   copy of this software and associated documentation files (the
+//   "Software"), to deal in the Software without restriction, including
+//   without limitation the rights to use, copy, modify, merge, publish,
+//   distribute, sublicense, and/or sell copies of the Software, and to
+//   permit persons to whom the Software is furnished to do so, subject to
+//   the following conditions:
+//
+//   The above copyright notice and this permission notice shall be included
+//   in all copies or substantial portions of the Software.
+//
+//   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+//   OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+//   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+//   IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+//   CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+//   TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+//   SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+namespace OxyPlot.DotNetCore.Drawing
+{
+   /// <summary>
+   /// The graphics render context.
+   /// </summary>
+   internal class GraphicsRenderContext : RenderContextBase, IDisposable
```

Comment:
The thing is I didn't want to depend on windows forms in my class as I wasn't writing a windows forms application.

Reply:
It's been a while since I worked on this and there are probably ways to reduce duplication of code. 

The biggest problem I ran into was the lack of support for a .net core implementation of System.Drawing.Common, I think. I made this pull request in December so my memory is a little fuzzy.

Reply:
I read the arguments against System.Drawing.Common and I agree but I also would need an alternative and it sounds like SkiaSharp is a good alternative. 

The application currently using this is running on an IIS server application.

What are your thoughts ?

Reply:
I suggest we keep the duplication for now and solve that later